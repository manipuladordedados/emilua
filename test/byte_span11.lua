local byte_span = require 'byte_span'

local bs = byte_span.new(1, 2):slice(-1)
print(#bs)
print(bs.capacity)
