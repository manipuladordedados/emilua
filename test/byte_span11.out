Main fiber from VM 0x1 panicked: 'ERANGE'
stack traceback:
	input.lua:1: in main chunk
	[C]: in function ''
	[string "?"]: in function <[string "?"]:0>
